scripts/Project.toml

```
@@ -2,4 +2,4 @@
 Clang = "40e3b903-d033-50b4-a0cc-940c62c95e31"
 
 [compat]
-Clang = "0.12"
\ No newline at end of file
+Clang = "0.12"
```

```suggestion
Clang = "0.17"
```
I didn't test, but try with this. It's what I needed for https://github.com/jump-dev/HiGHS.jl/pull/175

---

I tried this, without success so far. It's a different API for clang.jl, and produces a different file structure.

Do we need a generated Gurobi_JLL package as well to migrate to the new version?

---

Oh, we'd need to update the script as well: https://github.com/jump-dev/HiGHS.jl/tree/master/gen. No need for a JLL. And no real need to change. It's a low priority and you seem to have rebuild the wrapper okay.

---

Yep, I'll open another issue for that and post my attempt at the script changes.